### DISCUSSION TITLE
the data.y is always equal 1, why?

### DISCUSSION THREAD

**Discussion by jiaruHithub** [pyg-team/pytorch_geometric#5562] [Q&A] [answered]
I use dataset `BBBP` for generate some graph, I want to inherit the labels of the new graph directly from the existing graph, I use code:
```
for (class_idx, (label, graphs)) in zip(y_idxs, class_graphs):
        print(label)
        pyg_graph1 = Data()
        pyg_graph1.y = label
        print(pyg_graph1)
```
this dataset contains two classes, ```y_idxs``` contains graph idx for every class, ```class_graphs``` contains label and graph, and get output:
```
[-1]
Data(y=[1])
[1]
Data(y=[1])
```
Why does this happen? the first label of ```pyg_graph1``` I think is ```[-1]```, how can I fix this ?
Thanks!

**Comment by EdisonLeeeee** [comment, marked as answer]
Hi @jiaruHithub 

`Data(y=[1])` means that the size of `y` is `[1]`, a 1D tensor with length `1`, rather than the value `[1]`.

**Reply by jiaruHithub** [reply]
Thanks for edison again, its my slow
